jataware/dojo-deprecated
# fixed dummy_model for dojo, model-xform.py post_failed_to_dojo status…

### Description
- addresses issue #74 
- fixed dummy_model for dojo e.g. updated mapper.json
- `model-xform.py` `post_failed_to_dojo()` sets attribute `status` to `failed`.

### Testing
- pull this branch
- shutdown dojo containers 
  - remove all containers: `docker ps -q | % {docker stop $_}`
  - `docker container prune`
- start api
  - api$ `docker-compose -f docker-compose-dev.yaml up --build -d`
- start dmc
  - dmc$ `docker-compose up -d`
  - dmc$ `sudo chmod -R +777 logs mappers results model_configs`
- confirm with api/ health check at localhost:8000 (http://localhost:8000/healthcheck)
- create dummy-model
  - cd dojo/models/dummy_model
  - $`python3 Examples.py`
  - confirm with api /models
- create success run
  - copy dojo/models/dummy_model/TEST_RUN.json to api Runs/Create Run
  - give the id a unique prefix
  - submit and check in Airflow (http://localhost:8080)
  - confirm attributes status : success by api /runs/{run_id}
- create failure run
  - delete meta section from dojo/models/dummy_model/mapper.json
  - re-create dummy-model as above (ie. $python3 Examples.py)
  - create and check the run as above.
    - in api /runs/{run_id} attribute status should be failed
